### PR TITLE
fix(#71): naive fix excluding absolute path

### PR DESCRIPTION
**Pull request by T1p0un3t**

### CODE REVIEW

File: internal/application/config/Load.go [hunk 1]
```diff
@@ -70,7 +70,9 @@ func autocorrectValues(filename string, fileConfig *fileConfiguration) {
 	}
 
 	if specificationURLIsRelativeFilename(filename, fileConfig) {
-		fileConfig.OpenAPI.SpecificationURL = filepath.Dir(filename) + "/" + fileConfig.OpenAPI.SpecificationURL
+		if !strings.HasPrefix(fileConfig.OpenAPI.SpecificationURL, "/") {
```
Comment (strider2038) [inline internal/application/config/Load.go:73]: Hello! Thanks for PR. I think this check should be inside `specificationURLIsRelativeFilename` function.

```golang
func specificationURLIsRelativeFilename(filename string, fileConfig *fileConfiguration) bool {
	return filename != "" &&
		fileConfig.OpenAPI.SpecificationURL != "" &&
		!fileConfig.OpenAPI.urlFromEnv &&
		!strings.HasPrefix(fileConfig.OpenAPI.SpecificationURL, "/") &&
		!strings.HasPrefix(fileConfig.OpenAPI.SpecificationURL, "http")
}
```

Comment (T1p0un3t) [reply internal/application/config/Load.go:73]: Hello,

indeed it is easier to read and corresponds well to the need of the function.
Finaly the test is `strings.HasPrefix(fileConfig.OpenAPI.SpecificationURL, "./")`

Testing http and / is not useful just ./

```go
func specificationURLIsRelativeFilename(filename string, fileConfig *fileConfiguration) bool {
	return filename != "" &&
		fileConfig.OpenAPI.SpecificationURL != "" &&
		!fileConfig.OpenAPI.urlFromEnv &&
		strings.HasPrefix(fileConfig.OpenAPI.SpecificationURL, "./")
}
```

I have a question : in this case is prefixing with the directory a good solution? (`filepath.Dir(filename) + "/"`) ?

Comment (strider2038) [reply internal/application/config/Load.go:73]: > Testing http and / is not useful just ./

I agree.

> I have a question : in this case is prefixing with the directory a good solution? (filepath.Dir(filename) + "/") ?

I think if the path is relative, then it is expected to be relative to the configuration file, but not relative to the working directory. For example, if you run docker compose from another directory, all paths will be relative to the configuration file.

Comment (T1p0un3t) [reply internal/application/config/Load.go]: Ok, I hadn't thought of that scenario.
So perhaps the information on the documentation should be added to clarify this. I will create a ticket for this.

I updated the code to test if the path starts with ./ .

Thanks

**Pull request merged**